src/redux/saga/index.js

```diff
@@ -256,9 +256,17 @@ function* payPaypal({ payload }) {
 
 function* payByStripe({ payload }) {
     console.log(payload)
-    const res = yield axios.post(URL + apiEndPoint.STRIPE_PAY, { product: payload?.priceOfItem, User: payload?.currentUser, userAddress: payload?.userAddessData }, option)
-    payload?.handleStripeResponse(res)
-    console.log(res)
+    try {
+
+        const res = yield axios.post(URL + apiEndPoint.STRIPE_PAY, { product: payload?.priceOfItem, User: payload?.currentUser, userAddress: payload?.userAddessData }, option)
```

Comment:
instead of transforming the payload in saga , transform it in the confonent and then send payload here